src/content/api/merchants.mdx

```diff
@@ -58,8 +58,12 @@ which define the payment methods available for a Payment Request.
     Merchant [Settlement Config](#settlement-config-model).
   </Property>
 
+  <Property name="type" type="string">
+    Valid values: `physical` or `e-commerce`. Merchants with a physical store should provide type `physical`. Online only merchants should provide type `e-commerce`. E-commerce merchants will not be displayed in the store finder.
```

Comment:
Do you think we should specify if type isn't provided that it will default to 'physical'?

Reply:
That behaviour has changed, it will no longer default to `physical`.